spec/services/preserved_object_handler_update_version_spec.rb

```diff
@@ -536,10 +536,10 @@
             expect(pc.reload.last_checked_on_storage).to be > orig_timestamp
           end
           it 'ensures status of PreservedCopy is invalid' do
-            pc.status = Status.ok
+            pc.status = PreservedCopy.statuses[:ok]
             pc.save!
             po_handler.update_version_after_validation
-            expect(pc.reload.status).to eq Status.invalid_moab
+            expect(PreservedCopy.statuses[pc.reload.status]).to eq PreservedCopy.statuses[:invalid_moab]
```

Comment:
`PreservedCopy.statuses[pc.reload.status])` is gnarly syntax.  plain old `pc.reload.status` didn't work, eh?

Reply:
Yeah the `pc.reload.status` wasn't working 🤔

Reply:
i think the simpler way to do this would be:
```ruby
expect(pc.reload.status).to eq 'invalid_moab'`
```

because enum fields in active record objects return a string (though they accept the string, representation, the equivalent symbol, or the underlying integer).